specification/text_structure/test_grok_pattern/TestGrokPatternRequest.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RequestBase } from '@_types/Base'
+
+/**
+ * @rest_spec_name text_structure.test_grok_pattern
+ * @availability stack since=8.13.0 stability=experimental visibility=public
+ * @availability serverless stability=experimental visibility=private
```

Comment:
This should match the visibility of the find structure endpoint, as it will be used by the same UI functionality:

```suggestion
 * @availability stack since=8.13.0 stability=stable
 * @availability serverless stability=stable visibility=private
```

Reply:
fixed